src/base/bios/Makefile

```diff
@@ -3,12 +3,20 @@ top_builddir=../../..
 include $(top_builddir)/Makefile.conf
 
 CFILES=int10.c int16.c int17.c hlt.c setup.c vgabios.c bios_symbols.c
-SFILES=bios.S
 
 all: lib
 
 include $(REALTOPDIR)/src/Makefile.common
 
+bios.o: bios.S
```

Comment:
I reverted my recent change to move
configure bits to plugins.
Now you can just rename is to `bios.s`
(lowercase), and it will use the native
tool-chain. So this rule is not needed.